src/main/java/com/jungdam/diary/domain/vo/Comments.java

```diff
@@ -16,4 +19,14 @@
     public void add(Comment comment) {
         comments.add(comment);
     }
+
+    public void remove(Long id, Member member) {
+        for (Comment cmt : comments) {
+            if (cmt.isCreator(id, member)) {
+                comments.remove(cmt);
+                return;
+            }
+        }
+        throw new NotExistException(ErrorMessage.NOT_EXIST_COMMENT);
+    }
```

Comment:
depth를 줄이기위해 이러한 방식을 사용해보는 건 어떨까요?
```suggestion
    public void remove(Long id, Member member) {
        Comment target = comments.stream()
            .filter(c -> c.isCreator(id, member))
            .findFirst()
            .orElseThrow(() -> new NotExistException(ErrorMessage.NOT_EXIST_COMMENT));
        target.remove(cmt);
    }
```

Reply:
!!! 감사합니다! 수정 진행할게용!

Reply:
오 데이비 스트림 장인